pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -949,7 +949,7 @@ Channel channel() {
         return ctx.channel();
     }
 
-    SocketAddress serverAddrees() {
+    SocketAddress serverAddress() {
         return remoteAddress;
     }
```

Comment:
Seems this method isn't being used anywhere. Perhaps we could consider removing it.